Aws/DynamoDb/Core.hs

```diff
@@ -1348,6 +1349,19 @@ getAttr' k m = do
       Nothing -> return Nothing
       Just dv -> return $ fromValue dv
 
+-- | Combinator for parsing an attribute into a 'FromDynItem'.
+parseAttr
+    :: FromDynItem a
+    => T.Text
+    -- ^ Attribute name
+    -> Item
+    -- ^ Item from DynamoDb
+    -> Parser a
+parseAttr k m =
+  case M.lookup k m of
+    Nothing -> fail ("Key " <> T.unpack k <> " not found")
+    Just (DMap dv) -> either (fail "...") return $ fromItem dv
```

Comment:
It's certainly not :( Any idea how turn this into a sensible error message I've mentioned this here: https://github.com/aristidb/aws/pull/195#issuecomment-204200398, though that got lost I guess.